stacks-common/src/deps_common/bitcoin/blockdata/script.rs

```diff
@@ -201,8 +202,8 @@ fn build_scriptint(n: i64) -> Vec<u8> {
 
     let neg = n < 0;
 
-    let mut abs = if neg { -n } else { n } as usize;
-    let mut v = vec![];
+    let mut abs = n.abs() as usize;
+    let mut v = Vec::with_capacity(size_of::<usize>() + 2);
```

Comment:
Oh, good call, I almost missed that additional push, but it should be just 1, right? The loop handles all but the last byte, then the last byte gets handled, along with an optional extra byte for negative value.

Reply:
Oh, I didn't notice that that loop always leaves one byte